### DISCUSSION TITLE
Lineedit Focused overlay

### DISCUSSION THREAD

**Discussion by varlesh** [tsujan/Kvantum#668] [General]
Editable Lineedit interior can not be transparent, because overlay rename text.
Similar problem https://github.com/tsujan/Kvantum/issues/234
It's limit or can be fixed?
![image](https://user-images.githubusercontent.com/8083855/124416274-ae441700-dd67-11eb-9418-0de257bcb9ce.png)

**Comment by tsujan** [comment]
> Editable Lineedit interior can not be transparent

The limitation is imposed by Qt. No other style encounters it because no other style can have translucent SVG elements. Nothing can be done about it.

**Comment by varlesh** [comment]
Ok, thx... it's sadly :(
